src/compute/src/row_spine.rs

```diff
@@ -134,48 +132,28 @@ mod container {
 
         fn with_capacity(size: usize) -> Self {
             Self {
-                batches: vec![DatumBatch::with_capacities(size, size)],
+                bytes: BytesContainer::with_capacity(size),
```

Comment:
Oof, what does the size even mean? Anyway, not part of this PR, so ok to leave it as-is.

Reply:
No idea, but I think we should be in a position to make it mean something more concrete in the near future (next PR should involve right-sizing the allocations).